mkdocs/docs/api.md

```diff
@@ -1533,3 +1533,111 @@ df.show(2)
 
 (Showing first 2 rows)
 ```
+
+### Polars
```

Comment:
Thanks for the docs! This is great. Only thing i would add is to also mention the difference between
`iceberg_table.to_polars()` and `iceberg_table.scan().to_polars()`